### PR TITLE
Add Unipept taxonomy viewer plugin

### PR DESCRIPTION
**Pull request by caleb-easterly**
Adds a plugin that allows for visualization of taxonomy datasets. It is based on the visualizations at <http://unipept.ugent.be>, and relies on the standalone visualizations code from @unipept (at <https://github.com/unipept/unipept-visualizations>). The viewer appears only for the d3_hierarchy datatype (added by @jj-umn). 

This is a sample screenshot from the integrated Galaxy plugin, which opens in a separate tab: 
unipept_viz

The data is human gut microbiome data, Sample 7 from <a href="https://www.ncbi.nlm.nih.gov/pubmed/18971961"> Verberkmoes, et al. 2009</a>, also used on the Unipept website. 

The data in json format are also attached ([unipept-pept2lca.txt](https://github.com/galaxyproject/galaxy/files/1153477/unipept-pept2lca.txt)) - after upload, the `d3_hierarchy` datatype must be manually selected for the "Unipept Taxonomy Viewer" to appear in the vis drop-down menu.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "config/plugins/unipept"]
+	path = config/plugins/unipept
+	url = https://github.com/caleb-easterly/galaxy-plugin-unipept-visualization.git
```
Comment (jmchilton) [inline .gitmodules]: Thanks for the contribution! Did you mean for this to be here - the project as a submodule? It looks like you tried to add it as a submodule and then just added it directly - is that correct? I'd prefer not adding vizs as submodules to Galaxy for now - they'd be better just directly inserted into Galaxy IMO - at least for now.

Comment (caleb-easterly) [reply .gitmodules]: This was my first shot at working with submodules...and I mostly just got confused. So I deleted .gitmodules and the submodule file in .git. Would that clean it up enough?

File: config/plugins/visualizations/unipept/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
```
Comment (bgruening) [inline config/plugins/visualizations/unipept/LICENSE]: @caleb-easterly are you also fine to distribute this under the same license than the Galaxy project? If so you could remove this file as well.

Comment (caleb-easterly) [reply config/plugins/visualizations/unipept/LICENSE]: yep, just removed the license

**Comment by bgruening** [comment]
@caleb-easterly do we need the unipept-visualizations.min.js file with your latest commit?
Can you attach a simple screenshot to the PR description and maybe even an example file. I guess we will mention this addition in the Galaxy news brief and images are always great! :)

**Comment by caleb-easterly** [comment]
No, I think we just need the es5.js and es5.js.map, so I removed everything else. Screenshot and example file are in the PR description!

**Comment by bgruening** [comment]
Awesome stuff! @guerler ready to be merged?

**Comment by guerler** [comment]
Looks good to me.

**Pull request merged**